core/src/main/java/io/camunda/migrator/converter/DecisionInstanceConverter.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+package io.camunda.migrator.converter;
+
+import static io.camunda.migrator.impl.util.ConverterUtil.convertDate;
+import static io.camunda.migrator.impl.util.ConverterUtil.getNextKey;
+
+import io.camunda.db.rdbms.write.domain.DecisionInstanceDbModel;
+import io.camunda.migrator.impl.util.ConverterUtil;
+import io.camunda.search.entities.DecisionInstanceEntity;
+import java.util.List;
+import org.apache.commons.lang3.StringUtils;
+import org.camunda.bpm.engine.history.HistoricDecisionInputInstance;
+import org.camunda.bpm.engine.history.HistoricDecisionInstance;
+import org.camunda.bpm.engine.history.HistoricDecisionOutputInstance;
+
+public class DecisionInstanceConverter {
+
+  public DecisionInstanceDbModel apply(HistoricDecisionInstance decisionInstance,
+                                       Long decisionDefinitionKey,
+                                       Long processDefinitionKey,
+                                       Long decisionRequirementsDefinitionKey,
+                                       Long processInstanceKey,
+                                       Long rootDecisionDefinitionKey,
+                                       Long flowNodeInstanceKey,
+                                       String flowNodeId) {
+    Long decisionInstanceKey = getNextKey();
+    return new DecisionInstanceDbModel.Builder()
+        .partitionId(ConverterUtil.C7_HISTORY_PARTITION_ID)
+        .decisionInstanceId(String.format("%d-%s", decisionInstanceKey, decisionInstance.getId()))
+        .decisionInstanceKey(decisionInstanceKey)
+        .state(DecisionInstanceEntity.DecisionInstanceState.UNSPECIFIED) // TODO not stored in C7
```

Comment:
❓ If there's evaluation time, do we want to set it as EVALUATED per default? Or it will be incorrect if there's a failure?